test/parallel/common/assert.js

```diff
@@ -0,0 +1,6 @@
+function assert(thing, message) {
+  if (thing) return;
+  throw new Error(message);
+}
+
+return assert
```

Comment:
@harlanj Would you mind to add a blank line to the end of the file? It helps git not have some legacy strangeness. :)